java-client-api-tutorial/intro.md

```diff
@@ -0,0 +1,15 @@
+<p align="center">
+  <img src="assets/cyberark_logo.png">
+</p>
+
+# Introduction
+
+Welcome to an introduction of the Java client library for the Conjur API. In this tutorial, we will learn how to set up the Java client library for the Conjur API via a Kubernetes environment.
+
+## What is Conjur?
+Conjur is an open source interface to securely authenticate, control and audit non-human access across tools, applications, containers, and cloud environments via robust secrets management.
+
+*If you have never used Conjur before, we highly recommend that you [explore Conjur](https://www.conjur.org/) and read our documentation before starting this tutorial.*
+
+## What is the Java client library for the Conjur API?
+The Java client library provides a robust programmatic interface to a Conjur server from within your Java project. Use the Java SDK to authenticate with Conjur and fetch secrets in a secure manner. Integration with Conjur provides a variety of additional benefits including being able to store security policy as code, and automated secret rotation.
```

Comment:
I'd maybe just drop this section entirely. We can include links to relevant follow-up resources to learn more at the end. If we want more content on this main page, we could include something from step 1 that highlights the steps, but I think it'd be fine without it.